app/presenters/email_alert_presenter.rb

```diff
@@ -39,10 +39,25 @@ def standard_body(template_path)
         document_summary: document.summary,
         document_url: File.join(Plek.current.website_root, document.base_path),
         document_change_note: document.change_note,
+        document_org_title: document.org_title,
+        document_noun: document.document_noun,
+        updated_or_published: updated_or_published,
       }
     )
   end
 
+  def subject
+    publication_state ? document.title + " updated" : document.title
+  end
+
+  def updated_or_published
+    publication_state ? "updated" : "published"
+  end
+
+  def publication_state
+    document.publication_state == "redrafted"
+  end
```

Comment:
Would it be worth changing the name of this method to something like `#redrafted?` or flip the conditional and have `#published?`. I'd expected a method called `#publication_state` to return a string, whereas this is binary. What do you think?